app/components/affiliate-link-page/accept-referral-button.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="flex flex-col items-center justify-center" ...attributes>
```

Comment:
The layout logic here seems like something you'd want the caller to decide - so would remove this container element. You can pass `...attributes` directly to the `PrimaryButtonWithSpinner` component below.